WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -813,12 +813,8 @@ private void updateTitle() {
     }
 
     private void updateTitle(PageType pageType) {
```

Comment:
the `updateTitle()` method is no longer used so you can remove it

Reply:
Isn't it still used from `onPageChanged()`?

Reply:
I meant the one above this one, without parameters. I couldn't comment there :-)

Reply:
Done 👍